### PR TITLE
Garybao

### PR DESCRIPTION
**Pull request by Gary3321**
solved #3,4,6 issues

### CODE REVIEW

File: inst/CITATION [hunk 1]
```diff
@@ -1,19 +0,0 @@
-citHeader("To cite package ‘QFASA’ in publications use:")
```
Comment (justinkamerman) [inline inst/CITATION:1]: I don't think we should be deleting this file. I think the bug was to replace the paper referenced with a reference to the QFASA R package instead.

@cstewartGH can you confirm ?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,14 +1,16 @@
 Package: QFASA
 Title: Quantitative Fatty Acid Signature Analysis
 Version: 1.0.3
+Date: 2018-08-14
 Authors@R: c(
              person("Sara", "Iverson", email = "Sara.Iverson@dal.ca", role = c("aut")),
              person("Chris", "Field", email = "field@mathstat.dal.ca", role = c("aut")),
```
Comment (justinkamerman) [inline DESCRIPTION:7]: @cstewartGH is this the correct spelling for Chris Field ? I see there was a bug to fix the spelling of his name. I see it was Fields in the citiation.

**Review by justinkamerman — COMMENTED** [review]
Added a few questions for Connie here. Need to confirm a few things before I merge.

**Comment by cstewartGH** [comment]
Hongchang had asked me about deleting the file and I might have misunderstood, but I assumed there could only be one file for the citation and I assumed he meant that he was first deleting it and then replacing it.   If that’s not the case, someone might have to explain this to me. ☺

Chris Field is the proper spelling.  In the old citation (which referenced the paper not the package) it erroneously said “Chris Fields”.

**Review by justinkamerman — APPROVED** [review]
Approved.

**Pull request merged**